packages/runtime/src/is-node.tsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * Returns `true` when inside a node environment,
+ * else `false`.
+ *
+ * When using this it will remove any node code from the browser bundle - for example:
+ *
+ * ```js
+ * if (isNodeEnvironment()) {
+ *   // This code will be removed from the browser bundle
+ * }
+ * ```
+ */
+export const isNodeEnvironment = () => {
+  return (
+    Object.prototype.toString.call(typeof process !== undefined ? process : null) ===
+    '[object process]'
+  );
+};
```

Comment:
❤️

Reply:
This might be a little more resilient: 

```ts
const isNodeEnvironment = (): boolean => {
 typeof global !== 'undefined' && globalThis === global;
}
```

Reply:
Looks like `globalThis` is well supported: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis (note minimum node version of 12.x)

I worry that the current approach might break in a future node release